contentcuration/contentcuration/frontend/channelList/views/Channel/ChannelItem.vue

```diff
@@ -343,13 +345,21 @@
       }
     },
     methods: {
-      ...mapActions('channel', ['deleteChannel']),
+      ...mapActions('channel', ['deleteChannel', 'removeViewer']),
       ...mapMutations('channel', { updateChannel: 'UPDATE_CHANNEL' }),
       handleDelete() {
-        this.deleteChannel(this.channelId).then(() => {
-          this.deleteDialog = false;
-          this.$store.dispatch('showSnackbarSimple', this.$tr('channelDeletedSnackbar'));
-        });
+        if (!this.canEdit) {
+          const currentUserId = this.$store.state.session.currentUser.id;
+          this.removeViewer({ channelId: this.channelId, userId: currentUserId }).then(() => {
+            this.deleteDialog = false;
+            this.$store.dispatch('showSnackbarSimple', this.$tr('channelDeletedSnackbar'));
```

Comment:
Create a new string `channelRemovedSnackbar` with a value "Channel removed". The snackbar is currently displaying "Channel deleted" which is misleading in the context of a view-only" user.
<img width="1792" alt="Screenshot 2025-04-17 at 14 01 54" src="https://github.com/user-attachments/assets/91cccded-9d76-4683-a49d-1f51d96debaa" />